lib/orbit-common/cache.js

```diff
@@ -634,28 +635,29 @@ var Cache = Class.extend({
 
         record = this.retrieve(path);
         if (record.__rel) {
-          Object.keys(record.__rel).forEach(function(key) {
-            linkDef = _this.schema.models[type].links[key];
+          for (var m = 0, keys1 = Object.keys(record.__rel), len4 = keys1.length, key1; m < len4; ++m) {
```

Comment:
nit: super same as befores --- surely we can reuse some of these counters length names yeah?